CLI/model_loader.py

```
@@ -0,0 +1,37 @@
+from transformers import PreTrainedTokenizerFast
+from transformers import BartForConditionalGeneration
+
+from transformers import pipeline
+from transformers import T5ForConditionalGeneration,T5Tokenizer
+
+def load_kobart_summarizer():
+    tokenizer = PreTrainedTokenizerFast.from_pretrained('gogamza/kobart-summarization')
+    model = BartForConditionalGeneration.from_pretrained('gogamza/kobart-summarization')
+    return tokenizer, model
+
+from transformers import PreTrainedTokenizerFast, BartForConditionalGeneration
```

Ruff (0.8.2)

Redefinition of unused `PreTrainedTokenizerFast` from line 3

Remove definition: `PreTrainedTokenizerFast`

(F811)